docs/content/kitty-items/install.md

```diff
@@ -4,6 +4,11 @@ description: Set up the project locally
 sidebar_title: Install
 ---
 
+
+## Walkthrough Video
```

Comment:
nit: every other title is "Video Walkthrough"

Reply:
good catch, fixed!